### PR TITLE
Completed prep-javascript-variables

### PR DESCRIPTION
**Pull request by Edward-J-Kim**

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>JavaScript Variables</title>
+</head>
+
+<body>
+  <script>
+    var fullName = 'Edward Jaehoon Kim';
```
Comment (Zhiwutian) [inline prep-javascript-variables/index.html:10]: make sure to update all the `var` declarations to `const`

File: prep-javascript-variables/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    console.log('Value of fullName:', fullName);
+    console.log('Type of fullName:', typeof fullName);
+
+    var numberOfPets = 1;
+    console.log('Value of numberOfPets:', numberOfPets);
+    console.log('Type of numberOfPets:', typeof numberOfPets);
+
+    var likesVegetables = false;
+    console.log('Value of likesVegetables:', Boolean(likesVegetables));
```
Comment (Zhiwutian) [inline prep-javascript-variables/index.html:19]: you don't need to use the `Boolean` constructor here, just log the variable.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking good so far! a couple small updates are needed.

File: prep-javascript-variables/index.html (hunk 1 above)
Comment (Edward-J-Kim) [reply prep-javascript-variables/index.html]: update

updated the declarations

**Review by Zhiwutian — APPROVED** [review]
👍🏼

File: prep-javascript-variables/index.html (hunk 2 above)
Comment (Edward-J-Kim) [reply prep-javascript-variables/index.html]: fixed to log variable
fixed

**Pull request merged**